Develappers.BillomatNet/Mapping/InvoiceTagMapper.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using Develappers.BillomatNet.Api;
+using InvoiceTag = Develappers.BillomatNet.Types.InvoiceTag;
+using TagCloudItem = Develappers.BillomatNet.Types.TagCloudItem;
+
+
```

Comment:
Develappers.BillomatNet/Mapping/InvoiceTagMapper.cs#L12
The code must not contain multiple blank lines in a row. (SA1507)